craft_platforms/_architectures.py

```diff
@@ -70,6 +80,13 @@ def to_platform_arch(self) -> str:
         """
         return _ARCH_TRANSLATIONS_DEB_TO_PLATFORM.get(self.value, self.value)
 
+    def to_efi_arch(self) -> str:
+        """Convert this DebianArchitecture to an EFI string.
+
+        :returns: A string matching what os.uname().machine would return.
+        """
+        return _ARCH_TRANSLATIONS_DEB_TO_EFI.get(self.value, self.value)
```

Comment:
The combination of this and the dictionary below including RISC-V makes me unsure: what's the correct behaviour for S390X or ppc64el? (Especially the latter, some things call it `ppc64le`)